### DISCUSSION TITLE
Comments are enabled but no comments partial exists.

### DISCUSSION THREAD

**Discussion by worellana** [jpanther/congo#831] [Q&A]
Hi guys 

I'm getting a warning when using the Congo theme and enabling comments. I've read that the theme supports Hugo’s built-in Disqus comments. Furthermore, I've read the documentation, and it suggests adding a partial in the `layouts/partials/comments.html` path, however I've installed Congo using the Hugo Module, so if I add the file in the specified path it doesn't work. Could someone give me some light on what I'm doing wrong? I would really appreciate it.

The changes I've made to the config files are the following:

config\\_default\params.toml
```
showComments = true
```

config\\_default\config.toml
```
[services]
  [services.disqus]
    shortname = 'my_disqus_short_name'
```
Thanks in advance.

**Comment by jpanther** [comment]
As per the [Hugo docs](https://gohugo.io/content-management/comments/#render-hugos-built-in-disqus-partial-template), you need to add `{{ template "_internal/disqus.html" . }}` to your comments partial.

**Comment by worellana** [comment]
How can I add the comments partial if I have installed the Congo theme with Hugo module?

Sorry if the question is somehow dumb, I am getting used to Hugo
